src/SAMLTrace.js

```diff
@@ -405,6 +437,63 @@ SAMLTrace.RequestItem.prototype = {
     }
     target.appendChild(doc.createTextNode(samlFormatted));
   },
+  
```

Comment:
Instead of having the `showResponse` and `showRequest` tab you could just have a `showSummary` tab.

Within that combined tab you could write a headline by examining the `xmldoc`: If there's a `<saml2p:Response>`-node, it's a response, …, and so on. You get the point ;-)

Reply:
I like the idea of single tab instead of two different. Differentiation between those two resulted probably from that that at first I added showResponse and this was nice then I thought: lets add also info about request and I've added showRequest as separate thing instead of unifying both.